R/utils-shiny.R

```diff
@@ -0,0 +1,28 @@
+#' Internal helper for creating grid of `valuboxes`
+#' for displaying information.
+#'
+#' @keywords internal
+create_value_box_grid = function(df) {
```

Comment:
style: Lets pre-face the internal function with a `.`
```suggestion
.create_value_box_grid = function(df) {
```
Make sure to also apply this change whenever the function is called